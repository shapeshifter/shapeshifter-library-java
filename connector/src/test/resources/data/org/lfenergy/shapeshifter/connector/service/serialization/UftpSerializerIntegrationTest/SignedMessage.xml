<SignedMessage SenderDomain="piet.nl" SenderRole="AGR" Body="ZGl0IGlzIGVlbiBub2cKIGdlZW4gdGVrc3Q="/>