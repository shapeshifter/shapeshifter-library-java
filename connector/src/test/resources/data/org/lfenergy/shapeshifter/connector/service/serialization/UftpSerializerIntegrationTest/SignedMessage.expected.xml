<?xml version="1.0" encoding="UTF-8" standalone="yes"?>

<!--
Copyright 2023 Contributors to the Shapeshifter project

SPDX-License-Identifier: Apache-2.0
-->

<SignedMessage SenderDomain="piet.nl" SenderRole="AGR" Body="ZGl0IGlzIGVlbiBub2cKIGdlZW4gdGVrc3Q="/>